src/pages/MapPage/components/Mission/MissionDrawer/MissionStep2/MissionStep2.js

```diff
@@ -325,6 +397,67 @@ function MissionStep2() {
           }}
         />
       </Drawer>
+      <Drawer
+        anchor='bottom'
+        open={stateDrawer}
+        onClose={() => {
+          setStateDrawer(false)
+          setStateChoose(true)
+        }}
+        PaperProps={{
+          style: {
+            borderRadius: '20px 20px 0 0',
+            backgroundColor: '#FAFAFA',
+            height: '50vh'
+          }
+        }}
+      >
+        <Toolbar
+          style={{
+            position: 'sticky',
+            top: '0',
+            zIndex: '100',
+            backgroundColor: '#FAFAFA'
+          }}
+        >
+          <Typography variant='h6'>選擇狀態</Typography>
```

Comment:
Please also make this state required, and add remind before uploading to backend.